app/views/layouts/application.html.erb

```diff
@@ -117,8 +117,8 @@
       <div class="govuk-phase-banner feedback-banner">
         <p class="govuk-phase-banner__content">
          <span class="govuk-phase-banner__text">
-           This is a new service – your
-           <a class="govuk-link" href="mailto:govwifi-support@digital.cabinet-office.gov.uk?subject=User Feedback Suggestion">feedback</a> will help us to improve it.
+           This is a new service – your <%= link_to "feedback", help_index_path, class: "govuk-link" %>
+            will help us to improve it.
```

Comment:
Will this last part of the sentence render on a new line?

Reply:
Doesn't render a new line if it is within `<p>` tags. Its just on a new line to ensure the line length isn't too long on the screen.